### PR TITLE
[WNMDGS-2971] Change headers on CMSgov theme to Lexend

### PR DESCRIPTION
**Pull request by malloryiden**
## Summary

Follow up PR related to https://github.com/CMSgov/design-system/pull/3227.

Isolated only the header change for CMSgov.

## How to test

1. Confirm that Lexend was added as a design token.
2. Confirm that all headers are using Lexend.
3. Adding a little screenshot for reference:
image

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.
- [x] Selected appropriate release milestone

### CODE REVIEW

**Comment by malloryiden** [comment]
Q for @kim-cmsds Is this labeled with the correct release?

**Review by jack-ryan-nava-pbc — APPROVED** [review]
Checked locally and LGTM!

**Comment by kim-cmsds** [comment]
@malloryiden to answer your question, yes the milestone you selected is correct. However, we'll want to hold off on merging this until after #3276 is reviewed and merged. Right now, this branch is not rendering "Lexend" for headers on the docs site when looking at this locally. However, after cherrypicking (https://github.com/CMSgov/design-system/pull/3276/commits/6be25b15fb6029fda16bd10c42acacbfa8de816f), it does.

**Pull request merged**
